m4/internal/gnulib/config-openbsd.h

```diff
@@ -0,0 +1,232 @@
+#define GNULIB_SIGPIPE 1
+#define GNULIB_SNPRINTF 1
+#define GNULIB_STRERROR 1
+#define HAVE_ALLOCA 1
+#define HAVE_BTOWC 1
+#define HAVE_CANONICALIZE_FILE_NAME 1
+#define HAVE_DECL_ALARM 1
+#define HAVE_DECL_CLEARERR_UNLOCKED 0
+#define HAVE_DECL_FEOF_UNLOCKED 0
+#define HAVE_DECL_FERROR_UNLOCKED 1
+#define HAVE_DECL_FFLUSH_UNLOCKED 1
+#define HAVE_DECL_FGETS_UNLOCKED 1
+#define HAVE_DECL_FPURGE 0
+#define HAVE_DECL_FPUTC_UNLOCKED 0
+#define HAVE_DECL_FPUTS_UNLOCKED 0
+#define HAVE_DECL_FREAD_UNLOCKED 0
+#define HAVE_DECL_FSEEKO 1
+#define HAVE_DECL_FTELLO 1
+#define HAVE_DECL_FWRITE_UNLOCKED 0
+#define HAVE_DECL_GETCHAR_UNLOCKED 1
+#define HAVE_DECL_GETC_UNLOCKED 1
+#define HAVE_DECL_GETDTABLESIZE 1
+#define HAVE_DECL_ISBLANK 1
+#define HAVE_DECL_PROGRAM_INVOCATION_NAME 0
+#define HAVE_DECL_PROGRAM_INVOCATION_SHORT_NAME 0
+#define HAVE_DECL_PUTCHAR_UNLOCKED 1
+#define HAVE_DECL_PUTC_UNLOCKED 1
+#define HAVE_DECL_SIGALTSTACK 1
+#define HAVE_DECL_SNPRINTF 1
+#define HAVE_DECL_STRERROR_R 1
+#define HAVE_DECL_STRNDUP 1
+#define HAVE_DECL_STRNLEN 1
+#define HAVE_DECL_STRSIGNAL 1
+#define HAVE_DECL_SYS_SIGLIST 1
+#define HAVE_DECL__SNPRINTF 0
+#define HAVE_DECL___ARGV 0
+#define HAVE_DUP2 1
+#define HAVE_ENVIRON_DECL 1
+#define HAVE_FCNTL 1
+#define HAVE_FREXPL_IN_LIBC 1
+#define HAVE_FREXP_IN_LIBC 1
+#define HAVE_FSEEKO 1
+#define HAVE_GETCWD 1
+#define HAVE_GETDTABLESIZE 1
+#define HAVE_GETEGID 1
+#define HAVE_GETEUID 1
+#define HAVE_GETGID 1
+#define HAVE_GETOPT_H 1
+#define HAVE_GETOPT_LONG_ONLY 1
+#define HAVE_GETTIMEOFDAY 1
+#define HAVE_GETUID 1
+#define HAVE_INTMAX_T 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_INTTYPES_H_WITH_UINTMAX 1
+#define HAVE_ISBLANK 1
+#define HAVE_ISNAND_IN_LIBC 1
+#define HAVE_ISNANF_IN_LIBC 1
+#define HAVE_ISNANL_IN_LIBC 1
+#define HAVE_ISWCNTRL 1
+#define HAVE_ISWCTYPE 1
+#define HAVE_LANGINFO_CODESET 1
+#define HAVE_LANGINFO_H 1
+#define HAVE_LDEXPL_IN_LIBC 1
+#define HAVE_LDEXP_IN_LIBC 1
+#define HAVE_LIBINTL_H 1
+#define HAVE_LIMITS_H 1
+#define HAVE_LINK 1
+#define HAVE_LONG_LONG_INT 1
+#define HAVE_LSTAT 1
+#define HAVE_MALLOC_H 1
+#define HAVE_MALLOC_POSIX 1
+#define HAVE_MAP_ANONYMOUS 1
+#define HAVE_MATH_H 1
+#define HAVE_MBRTOWC 1
+#define HAVE_MBSINIT 1
+#define HAVE_MBSTATE_T 1
+#define HAVE_MBTOWC 1
+#define HAVE_MEMORY_H 1
+#define HAVE_MINMAX_IN_SYS_PARAM_H 1
+#define HAVE_MKDTEMP 1
+#define HAVE_MKSTEMP 1
+#define HAVE_MPROTECT 1
+#define HAVE_NL_LANGINFO 1
+#define HAVE_PIPE 1
+#define HAVE_PIPE2 1
+#define HAVE_POSIX_SPAWN 1
+#define HAVE_POSIX_SPAWNATTR_T 1
+#define HAVE_POSIX_SPAWN_FILE_ACTIONS_T 1
+#define HAVE_PTHREAD_MUTEX_RECURSIVE 1
+#define HAVE_PTHREAD_RWLOCK 1
+#define HAVE_RAISE 1
+#define HAVE_RAWMEMCHR 1
+#define HAVE_READLINK 1
+#define HAVE_REALPATH 1
+#define HAVE_SECURE_GETENV 1
+#define HAVE_SETRLIMIT 1
+#define HAVE_SIGACTION 1
+#define HAVE_SIGALTSTACK 1
+#define HAVE_SIGINTERRUPT 1
+#define HAVE_SIGSET_T 1
+#define HAVE_SIG_ATOMIC_T 1
+#define HAVE_SNPRINTF 1
```

Comment:
Setting this back to 0 will avoid a crash on this platform (see https://github.com/openbsd/ports/blob/master/devel/m4/patches/patch-lib_vasnprintf_c which handles this somewhat differently). This needs to be done on bison too.